MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import requests
+import json
+
+
+class ConfigManagerTest(unittest.TestCase):
```

Comment:
unittests need to be stand alone, repeatable and have no side effects.
here you're assuming the config manager is already running, and storing a new address on it.
you should start up a new config manager on the test class setup.
also, if the get_add_path test runs before the save, it would fail.
you should also test skipping the HTTP interface and test the methods/functions directly.